### ISSUE TITLE
Error if non ASCII encoding is used

### ISSUE THREAD

**Issue by PKaga** [gitandy/PyADIF-File#2]
I just test the import of ADIF file via pyadif. I have a ADIF file created by dxkeeper which contain 83k lines.
It throw an error - I reduced the lines in the adif file to 10k, 20k, 40k. The import is successfull with 10k and 20k lines in the adif file.

40k throw following error:

Traceback (most recent call last):
  File "C:\Users\HP 800 G1\PycharmProjects\hostilog 2022\read_adif.py", line 3, in <module>
    adi_doc = adi.load('df1lx-40k.adi')
  File "C:\Users\HP 800 G1\PycharmProjects\hostilog 2022\venv\lib\site-packages\adif_file\adi.py", line 159, in load
    data = af.read()
  File "C:\Program Files\Python39\lib\encodings\ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xef in position 12952561: ordinal not in range(128)

I may reduce to 30k lines but I believe the error happens around 32k lines.

73 de Peter DF1LX (df1lx at darc.de)

**Comment by PKaga** [comment]
I found a workaround. In the file adi.py the read line for the adif file should not be ascii or utf-8.

I changedd the source to

    with open(file_name) as af: # , encoding='utf-8') as af:
        dat = ""
        try:
            data = af.read()
        except Exception as e:
            print(e)
    return loads(data, skip)

and it works with 83000 QSO lines in the ADIF file. 
I do not see, what is wrong in my ADIF file as I created it just inbefore the check. There seems to be 2 or more challenges with e.g. "umlaute" or what else. As the file is 45 Megabytes it is "not easy" to find the "wrong" characters :)
